sdk/src/settings.rs

```diff
@@ -683,4 +683,44 @@ pub mod tests {
 
         reset_default_settings().unwrap();
     }
+
+    #[test]
+    fn test_all_setting() {
```

Comment:
That test is not related to that change, is it? It's a bonus test?

Reply:
yes this is bonus test. It was a handy way to document the current collection of settings in one place.